src/scss/global/_states.scss

```diff
@@ -0,0 +1,52 @@
+/** Component States */
+
+/*
+ * Disabled
+ *
+ * When the component is disabled, all interaction
+ * should be prevented. He we modify the bg color,
+ * and change the cursor displayed on the interactive
+ * components.
+ */
+
+$disabled-bg: rgb(248, 248, 248);
+$disabled-cursor: not-allowed;
+
+.vs--disabled {
+  .vs__dropdown-toggle,
+  .vs__clear,
+  .vs__search,
+  .vs__selected,
+  .vs__open-indicator {
+    cursor: $disabled-cursor;
+    background-color: $disabled-bg;
+  }
+}
+
+/*
+ *  RTL - Right to Left Support
+ *
+ *  Because we're using a flexbox layout, the `dir="rtl"`
+ *  HTML attribute does most of the work for us by
+ *  rearranging the child elements visually.
+ */
+
+.v-select[dir="rtl"] {
+  .vs__actions {
+    padding: 0 3px 0 6px;
```

Comment:
Now when we got postcss and the power of variables, I wonder if it would be a good idea to move some of these magical numbers into variables as well, there's a lot of `6px` padding/marging rules. Could it be worth declaring this once and giving it a suitable name?